readme.md

```diff
@@ -0,0 +1,225 @@
+The IEEE P370 Standard [(Electrical Characterization of Printed Circuit Board and Related Interconnects at Frequencies up to 50 GHz) ](https://ieeexplore.ieee.org/document/8078138)comes with an open source mathematical algorithm to achieve this goal. The source code can be found on the [IEEE gitlab](https://opensource.ieee.org/elec-char/ieee-370). Internally it uses a time domain transformation approach.
+
+Finally Figure 14 shows the prost processed measurement results from Figure 13 by applying Method 2 on the measured data.
+
+<p align = "center"><img src="./figures/plot_res_sim_vs_meas.png" alt="fig_plot_res_sim_vs_meas"  width="800"></p>
+<p align = "center">Figure 14: OpenMES Simulated S-Parameters vs. VNA-Measurement (Post-Processed/De-Embedded via Method 2) </p>
+
+We can see, that the worst case insertion loss around 17.5GHz is very close to our estimated value of -21.63dB which was calculated above. Overall the agreement is impressive considering the large manufacturing tolerances of a PCB fabricated by a prototyping service and a hand soldered connector.
+
+The connector itself has a very large return loss tolerance specification. From the datasheet:
+> VSWR = 1.1 + 0.02 x 20GHz = 1.5 (RL = 14dB)
+
+This is much worse than the measured result on its own! So, overall the agreement is remarkable.
+
+## 8. Conclusion
+
+To test the openEMS FDTD-Solver a SMA coaxial to microstrip transition was designed and simulated. 
+
+The Rosenberger 32K242-40ML5 connector has been integrated on a 6 layer test board with FR4 substrate fabricated by AISLER a german PCB prototyping service. Using castellated holes in a suitable configuration a PCB edge metallization has been realized with no additional cost. Finally a easy to use KiCAD footprint was created. The footprint enables the user to design complex RF circuits up to 20GHz on the AISER 6LayerHD FR4 stackup.  
+
+The transition is very low cost and intended for hobbyist and prototyping use. A 20GHz SMA transition for under 20€ as a "one off prototype order" is to my knowledge unique. 
+
+<div align="center">
+
+| Item                               | Cost          |
+|:-----------------------------------|:--------------|
+| 6 Layer PCB by AISLER (one piece)  | 20€/3 = 6.67€ |
+| Rosenberger connector (two pieces) | 2*6€ = 12€    |
+| Total Cost                         | 18.67€        |
+
+ </div>
+
+ The agreement between simulated and measured results is very good especially considering the high manufacturing tolerances due to hand soldering and a low cost PCB. OpenEMS does a remarkable job here! In my opinion this simulator gets way to less attention for its capabilities. 
+
+ This case study shows that it is possible to design high performance RF hardware using open source software only.
+
```

Comment:
I think a call-to-action might be nice here. Something along the lines:

- openEMS is awesome
- It is hard to use due to a lack of docs and teaching community
- This is a try to change this!
- -> Try to convince your boss to do internal research projects using openEMS and document your results.